frontend/demo/src/actions/messages/index.ts

```diff
@@ -0,0 +1,31 @@
+import {Dispatch} from 'redux';
+import {createAction} from 'typesafe-actions';
+import {HttpClient, Message, ResponseMetadataPayload} from 'httpclient';
+
+export const MESSAGES_LOADING = '@@messages/LOADING';
+
+export const loadingMessagesAction = createAction(
+  MESSAGES_LOADING,
+  resolve => (messagesInfo: {conversationId: string; messages: Message[]}) => resolve(messagesInfo)
+);
+
+export function listMessages(conversationId: string) {
+  return async (dispatch: Dispatch<any>) => {
+    return HttpClient.listMessages({
+      conversation_id: conversationId,
```

Comment:
the listMessages input type could respect JS conventions right? We could do the mapping inside the call to the backend? I'm wondering if the rest of the library looks like that and I didn't notice